src/main/java/com/db_access/entity/Letter.java

```diff
@@ -0,0 +1,71 @@
+package com.db_access.entity;
+
+import com.db_access.annotations.Reachable;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.validator.constraints.Length;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+public class Letter implements Serializable {
```

Comment:
Why it is _java.io.Serializable_ ?

Reply:
because in previous version I wrote that into file

Reply:
Now you do not need it.